app/assets/stylesheets/policies/_policy-voter-comparision.scss

```diff
@@ -129,4 +129,8 @@
 
 *:hover > .anchor {
   display: block;
-}
\ No newline at end of file
+}
+
+#col-sm-fixup {
```

Comment:
@Anishka0107 I think we need a different name than "fixup". Since we're always fixing things up, if we did this then pretty soon everything would be called "fixup"! :smile: 

Since it's an ID, perhaps describe what that element is?